File: GTTS/INL.VettingService.Models/AssignVettingBatch_Param.cs
using INL.Services.Models;

namespace INL.VettingService.Models
{
    public class AssignVettingBatch_Param : BaseParam, IAssignVettingBatch_Param
    {
        public long VettingBatchID { get; set; }
        public long AssignedToAppUserID { get; set; }
        public long ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/assign-vetting-batch_param.ts


import { IAssignVettingBatch_Param } from './iassign-vetting-batch_param';

export class AssignVettingBatch_Param implements IAssignVettingBatch_Param {
  
	public VettingBatchID: number = 0;
	public AssignedToAppUserID: number = 0;
  
	public ErrorMessages: string[] = null;
}


